### ISSUE TITLE
deployment instructions

### ISSUE THREAD

**Issue by vesparny** [vesparny/fair-analytics#7]
* nginx
* dockerfile

**Comment by joehand** [comment]
We deployed this with Docker! https://github.com/datproject/analytics-server. I'll try to send a PR with more detailed instructions.

There is some dat network issue we are still working on with docker, I'm going to debug that to see if I need to update the dockerfile.

**Comment by vesparny** [comment]
You use this on a datproject website??

This is super awesome and makes me happy 😊

**Comment by joehand** [comment]
> You use this on a datproject website??

Yes we are just getting it installed and trying it out! Would love to move away from third party trackers. 

Thanks for building it!

**Comment by vesparny** [comment]
Thanks to you for making dat 😀
Let me know if you have any problem setting it up.
Also your feedback would be highly appreciated
